bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```diff
@@ -189,7 +189,7 @@ spec:
               value: {{ .Values.persistence.mountPath | quote }}
             - name: PGDATA
               value: {{ printf "%s/%s" .Values.persistence.mountPath "data" | quote }}
-            {{- if and (or (not (include "postgresql-ha.postgresqlCreateSecret" .)) (include "postgresql-ha.postgresqlPasswordProvided" .)) (not (eq (include "postgresql-ha.postgresqlUsername" .) "postgres")) }}
+            {{- if and (or (not (include "postgresql-ha.postgresqlCreateSecret" .)) (include "postgresql-ha.postgresqlPostgresPassword" .)) (not (eq (include "postgresql-ha.postgresqlUsername" .) "postgres")) }}
```

Comment:
I don't understand the change `postgresql-ha.postgresqlPostgresPassword` doesn't return a boolean but the actual password which always has a value (provided or randomly created)

Reply:
I have updated the condition.
The `POSTGRES_POSTGRES_PASSWORD` env var should be generated when:
 - A secret is used -> use the provided
 - It is provided -> use the provided
 - When the user is not `postgres` -> generate a random or use the provided.